pkg/controller/drain.go

```diff
@@ -785,14 +785,14 @@ func (o *DrainOptions) evictPodWithoutPVInternal(attemptEvict bool, pod *corev1.
 		if err == nil {
 			break
 		} else if apierrors.IsNotFound(err) {
-			glog.V(3).Info("\t", pod.Name, " evicted")
+			glog.V(3).Info("\t", pod.Name, " evicted from node ", pod.Spec.NodeName)
```

Comment:
We need to capture the time taken for eviction/deletion of the pod here. We might have to introduce a time variable for the same.